src/lossrider/lossridersrc.py

```diff
@@ -400,3 +400,59 @@ def lossrider(
         return IFrame("https://www.linerider.com/", 900, 600)
     except ImportError:
         pass
+
+import wandb
+import pandas as pd
+from typing import List, Optional, Dict, Any
```

Comment:
Pls move these to the top of the file and omit wandb, I'd prefer not to add that as a dependency for all "users".
If we're being honest pandas should already have been a dependency so I'll accept that addition.